SPECS/reaper/reaper_build_caches.sh

```diff
@@ -0,0 +1,168 @@
+#! /bin/bash
+
+set -euxo pipefail
+
+# NOTE: EXECUTING with id=0 (root) CAUSES REAPER BUILD TO FAIL.
+if [[ $(id -u) -eq 0 ]]; then
+    echo "Must run script as non-root user!" >&2
+    exit 1
+fi
+
+# Cassandra reaper version for which to generate build caches.
+# (a) Modify VERSION="x.x.x" in script.
+# (b) Pass arguments to command line: ./reaper_build_caches.sh 3.1.1
+VERSION="3.1.1"
+if [ $? -gt 1 ]; then
+    VERSION=$1
+fi
+
+echo "Building Cassandra cache for version '$VERSION'."
+
+# URL to download the sources to build
+SOURCE_URL="https://github.com/thelastpickle/cassandra-reaper/archive/refs/tags/${VERSION}.tar.gz"
+
+# Build cache names
+BOWER_COMPONENTS="reaper-bower-components-${VERSION}.tar.gz"
+SRC_UI_NODE_MODULES="reaper-srcui-node-modules-${VERSION}.tar.gz"
+BOWER_CACHE="reaper-bower-cache-${VERSION}.tar.gz"
+MAVEN_CACHE="reaper-m2-cache-${VERSION}.tar.gz"
+NPM_CACHE="reaper-npm-cache-${VERSION}.tar.gz"
+LOCAL_LIB_NODE_MODULES="reaper-local-lib-node-modules-${VERSION}.tar.gz"
+LOCAL_N="reaper-local-n-${VERSION}.tar.gz"
+
+tempDir=$(mktemp -d)
+
+function cleanup {
+	exit_code=$?
+	set +e
+	if [[ -n $tempDir ]];then
+		rm -Rf $tempDir
+		echo "Deleted $tempDir."
+	fi
+	exit $exit_code
+}
+
+trap cleanup EXIT SIGINT SIGTERM
+
+mkdir -p ${tempDir}/mariner_caches
+
+marinerCacheDir=${tempDir}/mariner_caches
```

Comment:
Nit: reverting the order here will make it easier to see that the created directory and `marinerCacheDir` are the same thing (the reader doesn't have to compare the paths themselves).
```suggestion
marinerCacheDir=${tempDir}/mariner_caches

mkdir -p ${marinerCacheDir}
```

Reply:
Done